### ISSUE TITLE
Use IIIF for thumbnails?

### ISSUE THREAD

**Issue by terrellt** [OregonDigital/oregondigital_2#189]

**Comment by jechols** [comment]
We should figure out all the pros/cons here - we save on disk and derivative generation for sure, but it would add a not insignificant load to IIP.  From our (admittedly limited) tests on the Historic Oregon Newspaper project, thumbnails were pretty close in terms of CPU load versus tiles.

It might be worth caching IIP via memcached if we go this route.

**Comment by srabun** [comment]
yes but do we have the resources? is it worth the extra cpu hit? no thumbs in fedora right now, serving straight from disk.

**Comment by srabun** [comment]
let's try it

**Comment by terrellt** [comment]
Document thumbnails may be a problem.
